scripts/ci-e2e.sh

```diff
@@ -89,7 +89,7 @@ source "${REPO_ROOT}/hack/ensure-kustomize.sh"
 
 # This will run the tests with env variabls defined in environment.sh
 # or exported by metal3-dev-env scripts
-${M3_DEV_ENV_PATH}/scripts/run_command.sh make e2e-tests
+${M3_DEV_ENV_PATH}/scripts/run_command.sh make e2e-tests || sleep infinity
```

Comment:
Why should we have sleep infinity here?

Reply:
Oops ! used temporarily for debug, removed thanks